big_transfer.py

```diff
@@ -94,4 +134,4 @@ def encode(self, docs: DocumentArray, **kwargs) -> DocumentArray:
         output = _output['output_1'].numpy()
         for index, doc in enumerate(docs):
             doc.embedding = output[index]
-        return docs
\ No newline at end of file
+        return docs
```

Comment:
do we have traversal path for this encoder?

Reply:
Implemented it just now: https://github.com/jina-ai/executor-big-transfer-encoder/pull/1/commits/0a03a3dc9d13f41ed3093dc6d8e1be64e4c60586